src/scenes/CreateTopic/components/createTopic.js

```diff
@@ -145,23 +216,23 @@ class CreateTopic extends React.Component {
                   onClick={this.onAddResultField}
                   style={{ width: WIDTH_RESULT_FIELD, marginBottom: '32px' }}
                 >
-                  <Icon type="plus" /><FormattedMessage id="create.addresult" />
+                  <Icon type="plus" /><FormattedMessage id="create.addresult" defaultMessage="Add Result" />
                 </Button>
               ) : null}
             </FormItem>
           </FormItem>
 
           <FormItem
             {...formItemLayout}
-            label={this.props.intl.formatMessage({ id: 'create.resultsetter' })}
-            extra={this.props.intl.formatMessage({ id: 'create.resultsetterextra' })}
+            label={<FormattedMessage id="create.resultsetter" defaultMessage="Centralized Oracle" />}
+            extra={<FormattedMessage id="create.resultsetterextra" defaultMessage="This person will set the result." />}
```

Comment:
how come this is not using the `messages.xxx` format?

Reply:
is there a reason to use `FormattedMessage` vs `intl.formatMessage`?

Reply:
`messages.xxx` is for the intl.formatMessage, 
`FormattedMessage` is a component and `intl.formatMessage` is the API from react intl
it's recommended to use `FormattedMessage` for most of the possible situations
but some string cannot be used with the `FormmatedMessage` like placeholder, at this time we use intl.formatMessage.

Reply:
gotcha. thanks